src/main/webapp/app/course/dashboards/assessment-dashboard/assessment-dashboard-information.component.ts

```diff
@@ -33,8 +37,12 @@ export class AssessmentDashboardInformationEntry {
     selector: 'jhi-assessment-dashboard-information',
     templateUrl: './assessment-dashboard-information.component.html',
     styleUrls: ['./assessment-dashboard-information.component.scss'],
+    standalone: true,
+    imports: [TranslateDirective, PieChartModule, ArtemisSidePanelModule, RouterLink, ArtemisTranslatePipe],
 })
 export class AssessmentDashboardInformationComponent implements OnInit, OnChanges, OnDestroy {
+    private translateService = inject(TranslateService);
+
     @Input() isExamMode: boolean;
```

Comment:
Might be feasible to also do the input() migration here?

Reply:
Signals migration will be done in a follow up :)